tools/cldr-code/src/main/java/org/unicode/cldr/util/VoterReportStatus.java

```diff
@@ -35,5 +55,43 @@ public ReportStatus mark(ReportId r, boolean asComplete, boolean asAcceptable) {
             }
             return this;
         }
+
+        /**
+         * Return the acceptability enum for this report type
+         * @param r report type
+         * @return the acceptability enum, or null if there was no entry
+         */
+        public ReportAcceptability getAcceptability(ReportId r) {
+            return ReportAcceptability.fromPair(completed.contains(r), acceptable.contains(r));
+        }
+    }
+
+    /**
+     * Update a Resolver for a particular Report. The resolver will be cleared.
+     * Note that T must be an Integer for this to succeed.
+     * @param l locale
+     * @param r which report
+     * @param userList set of users
+     * @param status the source
+     * @param res which
+     * @return resolver
+     */
+    public VoteResolver<ReportAcceptability>
```

Comment:
ordinarily we have "VoteResolver<String>" where the strings are candidate items...

is the plan to use "VoteResolver<ReportAcceptability>" to resolve votes between "acceptable" and "notAcceptable"?

Reply:
yes, that's exactly what it does.  it's a voteresolver applied to an enum.

Reply:
it means that all the usual voting rules apply- per-organization, strengths, etc.